Specifications/QIR/Callables.md

```diff
@@ -291,7 +287,7 @@ have access to the controlled count, and so can't unambiguously determine the
 expected argument tuple; specifically, it can't tell if an inner tuple is the
 result of an application of `Controlled` or just part of the base signature.
 
-#### Implementing Lambdas
+## Implementing Lambdas, Partial Application, and Currying
 
 The language-specific compiler should generate a new top-level callable of the
 appropriate type with implementation provided by the anonymous body of the lambda;
```

Comment:
(These lines are not a part of a change in this PR, that is why feel free to ignore. Just in case I would like to make sure that you have this in mind)

Since this is a part of the public documentation, the external readers may have the same questions that we have. If we don't resolve those questions beforehand, then multiple users can ask us the same questions.

> should generate a new **top-level** callable

I'm not sure I correctly get the point of "top-level" fragment here. Is it a "global scope" callable? (as opposed to the _local scope_ callable)

Reply:
Yes, edited to say "callable at the global scope".